packages/dotenv-linter/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@audius/dotenv-linter",
+  "private": true,
+  "version": "1.5.67",
+  "description": "dotenv lint",
+  "author": "Audius",
+  "homepage": "https://github.com/AudiusProject/audius-protocol#readme",
+  "license": "Apache-2.0",
+  "bin": {
+    "dotenv-linter": "bin/dotenv-linter"
+  },
+  "scripts": {
+    "install": "BINDIR=./bin curl -sSfL https://raw.githubusercontent.com/dotenv-linter/dotenv-linter/v3.3.0/install.sh | sh -s"
```

Comment:
just wondering if there's a toolchain or package manager way to do this that isn't piping a downloaded shell script? Though I would not be surprised if many packages we are installing do a similar thing

Reply:
I looked into this for a bit... didn't find much :(  but I also think this is reasonable in terms of how npm handles things, except we're pulling from github rather than npm's cdn.